promql/promqltest/testdata/functions.test

```diff
@@ -6,18 +6,15 @@ load 5m
 
 # Tests for resets().
 eval instant at 50m resets(http_requests[5m])
-	{path="/foo"} 0
-	{path="/bar"} 0
-	{path="/biz"} 0
 
 eval instant at 50m resets(http_requests[20m])
 	{path="/foo"} 1
 	{path="/bar"} 0
 	{path="/biz"} 0
 
 eval instant at 50m resets(http_requests[30m])
-	{path="/foo"} 2
-	{path="/bar"} 1
+	{path="/foo"} 1
+	{path="/bar"} 0
 	{path="/biz"} 0
```

Comment:
In a similar spirit, let's add a 32m range to demonstrate the two counter resets for `/foo`.